crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -54,17 +54,8 @@ fn derive_bundle_(input: DeriveInput) -> Result<TokenStream2> {
         }
     };
     let (tys, field_members) = struct_fields(&data.fields);
-    let manifest = Manifest::new().unwrap();
-    let path_str = if let Some(package) = manifest.find(|name| name == "bevy") {
-        format!("{}::ecs", package.name)
-    } else if let Some(package) = manifest.find(|name| name == "bevy_internal") {
-        format!("{}::ecs", package.name)
-    } else if let Some(package) = manifest.find(|name| name == "bevy_ecs") {
-        package.name
-    } else {
-        "bevy_ecs".to_string()
-    };
-    let crate_path: Path = syn::parse(path_str.parse::<TokenStream>().unwrap()).unwrap();
+
+    let crate_path = bevy_ecs_path();
```

Comment:
Maybe `ecs_crate` or even just `ecs` would be better?

Reply:
Changed it to `ecs`.

Reply:
imo removing "path" from the name makes it harder to tell what the variable is. can we do `ecs_path`?

Reply:
or alternatively ecs_crate?

Reply:
I changed it to `ecs_path`.